### ISSUE TITLE
Intellisence not working

### ISSUE THREAD

**Issue by detautama** [lightyen/vscode-tailwindcss-twin#9]
I have a problem with intellisence in v0.8.1 

![MicrosoftTeams-image](https://user-images.githubusercontent.com/18025983/113253737-43aeb580-92f8-11eb-9047-762359b01212.png)

**Comment by lightyen** [comment]
Could you give me the content of `tailwind.config.js` or a repo link? Thanks.

**Comment by detautama** [comment]
tailwind.config.js

```
/* eslint-disable @typescript-eslint/no-var-requires */
const tailwindColors = require('tailwindcss/colors')

const colors = {
  /* Add custom colors here */
}

module.exports = {
  purge: {
    enabled: true,
    content: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
  },
  theme: {
    extend: {
      colors: {
        ...colors,
        primary: {
          light: tailwindColors.indigo[500],
          DEFAULT: tailwindColors.indigo[600],
          dark: tailwindColors.indigo[700],
          contrast: tailwindColors.white,
        },
        secondary: {
          light: tailwindColors.pink[500],
          DEFAULT: tailwindColors.pink[600],
          dark: tailwindColors.pink[700],
          contrast: tailwindColors.white,
        },
        success: {
          light: tailwindColors.emerald[500],
          DEFAULT: tailwindColors.emerald[600],
          dark: tailwindColors.emerald[700],
          contrast: tailwindColors.white,
        },
        error: {
          light: tailwindColors.red[500],
          DEFAULT: tailwindColors.red[600],
          dark: tailwindColors.red[700],
          contrast: tailwindColors.white,
        },
      },
      fontFamily: {
        display: ['Inter', 'sans-serif'],
        body: ['Inter', 'sans-serif'],
      },
    },
  },
  variants: {
    extend: {
      backgroundColor: ['checked'],
      borderColor: ['checked'],
      inset: ['checked'],
      zIndex: ['hover', 'active'],
    },
  },
  plugins: [require('@tailwindcss/forms')],
  future: {
    purgeLayersByDefault: true,
  },
}

```

**Comment by lightyen** [comment]
Fixed in v0.8.2
